src/lib/pubkey/sphincsplus/sphincsplus_common/sp_parameters.h

```diff
@@ -18,7 +18,7 @@ namespace Botan {
 enum class Sphincs_Hash_Type {
    Shake256,
    Sha256,
-   Haraka,  ///< Haraka is currently not supported
+   BOTAN_DEPRECATED("Haraka is not and will not be supported") Haraka,  ///< Haraka is currently not supported
```

Comment:
When using SLH-DSA, I get a syntax error since `[[deprecated]]` needs to be after the enum element

```suggestion
   Haraka BOTAN_DEPRECATED("Haraka is not and will not be supported"),  ///< Haraka is currently not supported
```

Reply:
True! Thank you!